decidim-meetings/db/migrate/20201009093140_add_registration_type_and_url_to_meetings.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddRegistrationTypeAndUrlToMeetings < ActiveRecord::Migration[5.2]
+  def change
+    add_column :decidim_meetings_meetings, :registration_type, :string
```

Comment:
Since the `registration_type` is mandatory I'd add a `null: false` constraint. The problem is if the platform already has meetings we need to set a default value as well 😄 . What do you think?

Reply:
Yes, that makes sense. Should I put default option as "on_this_platform" or "registrations_disabled"?

Reply:
I'm not really sure about that! Summoning @mrcasals 😄

Reply:
I'd put "registrations disabled" by default, and then update all official meetings to set the value to "on_this_platform"